node/sc/mainbridge_test.go

```diff
@@ -38,12 +38,14 @@ import (
 	"github.com/klaytn/klaytn/storage/database"
 	"github.com/stretchr/testify/assert"
 	"math/big"
+	"path"
 	"reflect"
 	"strings"
 	"testing"
 )
 
 const testNetVersion = uint64(8888)
+const testDataDir = "./tmp_bridge_test"
```

Comment:
How about use this pattern?
```
	tempDir, err := ioutil.TempDir(os.TempDir(), "sc")
	assert.NoError(t, err)
	defer func() {
		if err := os.RemoveAll(tempDir); err != nil {
			t.Fatalf("fail to delete file %v", err)
		}
	}()
```

Reply:
Thank you. I reflected it. Check this again.